tasmota/language/de_DE.h

```diff
@@ -526,7 +526,7 @@
 // xdrv_89_dali.ino
 #define D_SENSOR_DALI_RX                  "Dali RX"
 #define D_SENSOR_DALI_TX                  "Dali TX"
-#define D_CONFIGURE_DALI                  "Konfig․ DALI"
+#define D_CONFIGURE_DALI                  "DALI Einstellungen"
```

Comment:
DALI-Einstellungen same as KNX-Einstellungen?

Reply:
Do you mean the hyphen?

Reply:
yes, with or without the hyphen.

Reply:
Thanks for the hint.